frontend/src/metabase/query_builder/actions/parameters.ts

```diff
@@ -0,0 +1,24 @@
+import { CardApi } from "metabase/services";
+import { Parameter } from "metabase-types/api";
+import Question from "metabase-lib/Question";
+
+interface FetchParameterValuesOpts {
+  question: Question;
+  parameter: Parameter;
+  query?: string;
+}
+
+export const fetchQuestionParameterValues =
+  ({ question, parameter, query }: FetchParameterValuesOpts) =>
+  async () => {
+    const apiArgs = { cardId: question.id(), paramId: parameter.id, query };
+
+    const { values, has_more_values } = query
+      ? await CardApi.parameterSearch(apiArgs)
+      : await CardApi.parameterValues(apiArgs);
```

Comment:
While looking at QB code, I expect `query` to be an instance of a `StructuredQuery` or a `NativeQuery`. It doesn't seem to be the case here though. Is there a different name we can use? I literally was trying to figure out why the way we choose an endpoint depends on the presence of a question query 😅